rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_GCC/TARGET_M33/irq_armv8mml_fp.S

```diff
@@ -1,3 +0,0 @@
-        .equ     __FPU_USED,  1
```

Comment:
we do not need these removed files at all?

Reply:
No, we don’t need them. We can add FPU and other defines from tools, based on core selected or using -D option

Reply:
Let's not use -D if we can avoid it.

Reply:
Ideally FPU and DSP selection should be supported from tools, agree to that.